MC/config/common/external/generator/TPCLoopers.C

```diff
@@ -377,6 +379,11 @@ class GenTPCLoopers : public Generator
 } // namespace eventgen
 } // namespace o2
 
+// ONNX model files can be local, on AliEn or in the ALICE CCDB.
+// For local and alien files it is mandatory to provide the filenames, for the CCDB instead the
+// path to the object in the CCDB is sufficient. The model files will be downloaded locally.
+// Example of CCDB path: "ccdb:Users/n/name/test"
```

Comment:
I have some doubts about the prefix `ccdb:`. Is this already used like this somewhere else? I see your intention but maybe we could stick to the more common practice of having protocols be specified with `ccdb://` (similar to `alien://` or `http://`). I am also wondering if there are actually user paths like `Users/n/name` on our CCDB (so far I have not found any)?